src/AspNet.Security.OAuth.Alipay/RSAHelper.cs

```diff
@@ -0,0 +1,317 @@
+﻿using System;
```

Comment:
The need for this entire class here concerns me. We the maintainers don't really want to look after all this cryptographic code that _may_ contain security vulnerabilities on an ongoing basis.

It's also non-trivial, and documented in Mandarin/Cantonese, which none of the maintainers speak.